tests/unit/backend/corpora/api_server/base_api_test.py

```diff
@@ -48,7 +53,69 @@ def _remove_timestamps(jrb, removing):
         return _remove_timestamps(body, remove_attributes)
 
 
+def mock_assert_authorized_token(token: str, audience: str = None):
+    if token == "owner":
+        return {"sub": "test_user_id", "email": "fake_user@email.com", "scope": []}
+    elif token == "not_owner":
+        return {"sub": "someone_else", "email": "fake_user@email.com", "scope": []}
+    elif token == "super":
+        return {"sub": "super", "email": "fake_user@email.com", "scope": ["write:collections"]}
+    else:
+        raise Exception()
+
+
+def get_cxguser_token(user="owner"):
+    """
+    Generated an auth token for testing.
+    :param app: a WSGI app.
```

Comment:
nit: this line should be updated to `user`

Reply:
```suggestion
    :param user: the type of use the token will simulate.
```